### PR TITLE
Round the outstanding balance for correct coloring

### PR DESCRIPTION
**Pull request by SimonaGerbocova**
Issue: color of the outstanding balance value if it equals 0.00 €
Some people see it displayed in green color, some people in red color (because value isn't rounded to 2 decimal places, but only displayed in eur format).

### CODE REVIEW

File: client/src/pages/PayrollPage.js [hunk 1]
```diff
@@ -77,11 +77,13 @@ class PayrollPage extends Component {
 
     const balance = _.sum(filteredData.map((v) => v.amount))
 
-    const formattedBalance = eurFormat(balance)
+    const roundedBalance = parseFloat(balance.toFixed(2))
```
Comment (paldys) [inline client/src/pages/PayrollPage.js]: Let's use `Math.round(balance * 100) / 100` for rounding.

File: client/src/pages/PayrollPage.js [hunk 2]
```diff
@@ -63,7 +63,7 @@ class PayrollPage extends Component {
     return (
       <Fragment>
         Selected {selectedAmounts.length} items of total value&nbsp;
-        <span style={{color: inFlow + outFlow > 0 ? color.positive : color.negative, fontWeight: 'bold'}}>
+        <span style={{color: inFlow + outFlow !== 0 ? (inFlow + outFlow > 0 ? color.positive : color.negative) : color.defaultText, fontWeight: 'bold'}}>
```
Comment (paldys) [inline client/src/pages/PayrollPage.js]: On line 84, you use the same logic to decide about the color. Can we extract that as a function and use it here too?

Comment (paldys) [inline client/src/pages/PayrollPage.js]: `inFlow + outFlow !== 0` will have the same problem as balance. Won't it?

File: client/src/pages/PayrollPage.js [hunk 3]
```diff
@@ -77,11 +77,13 @@ class PayrollPage extends Component {
...
     const balance = _.sum(filteredData.map((v) => v.amount))
 
-    const formattedBalance = eurFormat(balance)
+    const roundedBalance = parseFloat(balance.toFixed(2))
 
-    const textColor = balance !== 0 ? (balance > 0 ? 'positive' : 'negative') : 'defaultText'
+    const formattedBalance = eurFormat(roundedBalance)
 
-    const balanceNote = balance > 0 ? '(Vacuumlabs owes you)' : '(You owe Vacuumlabs)'
+    const textColor = roundedBalance !== 0 ? (roundedBalance > 0 ? 'positive' : 'negative') : 'defaultText'
```
Comment (hleumas) [inline client/src/pages/PayrollPage.js]: No need to do any rounding, just do
```js
  balance >= 5e-3
? positive
: balance <= 5e-3
? negative
: 'default'

Comment (SimonaGerbocova) [reply client/src/pages/PayrollPage.js]: thank you for your simple solution :)

File: client/src/pages/PayrollPage.js [hunk 4]
```diff
@@ -52,6 +52,18 @@ class PayrollPage extends Component {
     return data.filter((item) => isWithinRange(item.date, dateRange))
   };
 
+  renderTextColor = (amount) => {
+    const textColor = amount >= 5e-3 ? 'positive' : amount <= -5e-3 ? 'negative' : 'defaultText'
+
+    return textColor
```
Comment (paldys) [inline client/src/pages/PayrollPage.js]: This can be a one-liner. :slightly_smiling_face:

File: client/src/pages/PayrollPage.js [hunk 5]
```diff
@@ -52,6 +52,18 @@ class PayrollPage extends Component {
     return data.filter((item) => isWithinRange(item.date, dateRange))
   };
 
+  renderTextColor = (amount) => {
```
Comment (paldys) [inline client/src/pages/PayrollPage.js]: This is more of a `getTextColor`.

File: client/src/pages/PayrollPage.js [hunk 6]
```diff
@@ -79,20 +93,18 @@ class PayrollPage extends Component {
 
     const formattedBalance = eurFormat(balance)
 
-    const textColor = balance !== 0 ? (balance > 0 ? 'positive' : 'negative') : 'defaultText'
-
-    const balanceNote = balance > 0 ? '(Vacuumlabs owes you)' : '(You owe Vacuumlabs)'
```
Comment (paldys) [inline client/src/pages/PayrollPage.js]: It was unnecessary to extract this.

Let's go with:
```
     const balanceNote = balance >= 5e-3 ? '(Vacuumlabs owes you)' : balance <= -5e-3 ? '(You owe Vacuumlabs)' : null
```

..and on line 106 you can:
```
 {(balanceNote && (
          <span className={classes.note}>{balanceNote}</span>
)}
```

File: client/src/pages/PayrollPage.js (hunk 1 above)
Comment (SimonaGerbocova) [reply client/src/pages/PayrollPage.js]: I used a different solution and therefore rounding is unnecessary.

File: client/src/pages/PayrollPage.js (hunk 2 above)
Comment (SimonaGerbocova) [reply client/src/pages/PayrollPage.js]: Yes, I created a function for that.

File: client/src/pages/PayrollPage.js (hunk 4 above)
Comment (SimonaGerbocova) [reply client/src/pages/PayrollPage.js]: fixed in the last commit :)

File: client/src/pages/PayrollPage.js (hunk 5 above)
Comment (SimonaGerbocova) [reply client/src/pages/PayrollPage.js]: fixed in the last commit :)

File: client/src/pages/PayrollPage.js (hunk 6 above)
Comment (SimonaGerbocova) [reply client/src/pages/PayrollPage.js]: fixed in the last commit :)

**Pull request merged**